src/routes/embedded/account/backup-wallet/backup-wallet-copy-seedphrase.view.tsx

```diff
@@ -36,114 +37,112 @@ export function AccountBackupCopySeedphraseEmbeddedView() {
-              position: "absolute",
-              right: 8,
-              bottom: 8,
-              maxWidth: "fit-content",
-            }}
-            onClick={() => setIsVisible(!isVisible)}>
-            {isVisible ? (
-              <EyeOff
-                style={{
-                  height: 16,
-                  width: 16,
-                  color: "#000",
-                }}
-              />
-            ) : (
-              <Eye
-                style={{
-                  height: 16,
-                  width: 16,
-                  color: "#000",
-                }}
-              />
-            )}
-          </div>
-          {isLoading && (
-            <div
-              style={{
-                position: "absolute",
-                top: 0,
-                left: 0,
-                right: 0,
-                bottom: 0,
-                display: "flex",
-                alignItems: "center",
-                justifyContent: "center",
-              }}>
-              <Loading color={"#000"} />
-            </div>
-          )}
+            {seedphrase}
+          </Text>
         </Box>
-        <Flex
-          gap={8}
-          align="center"
-          justify="center"
+
+        <div
           style={{
             cursor: "pointer",
-            marginTop: -20,
-            marginLeft: 8,
-            alignSelf: "flex-start",
+            position: "absolute",
+            right: 8,
+            bottom: 8,
+            maxWidth: "fit-content",
           }}
-          onClick={copySeedphrase}>
-          <Text variant="bodyMd">{isCopied ? "Copied" : "Copy"}</Text>
-          {isCopied ? (
-            <CheckIcon style={{ height: 16, width: 16, color: "#22c55e" }} />
+          onClick={() => setIsVisible(!isVisible)}>
+          {isVisible ? (
+            <EyeOff
+              style={{
+                height: 16,
+                width: 16,
+                color: "#000",
+              }}
+            />
           ) : (
-            <Copy01 style={{ height: 16, width: 16 }} />
+            <Eye
+              style={{
+                height: 16,
+                width: 16,
+                color: "#000",
+              }}
+            />
           )}
-        </Flex>
-        <Box style={{ padding: 0 }}>
-          <Button isFullWidth onClick={() => navigate("/wallet")}>
-            Done
-          </Button>
-        </Box>
+        </div>
+
+        {isLoading && (
+          <div
+            style={{
+              position: "absolute",
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0,
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
+            }}>
+            <Loading color={"#000"} />
+          </div>
+        )}
       </Box>
-    </Card>
+
+      <Flex
+        gap={8}
+        align="center"
+        justify="center"
+        style={{
+          cursor: "pointer",
+          marginTop: -20,
```

Comment:
vertical gap missing between the seedphrase and the Copy text and is related to `marginTop`:
![image](https://github.com/user-attachments/assets/f01d6386-96da-4c46-b5ee-983d12b4498e)

Reply:
I've created a new input field to address the different issues here: inconsistent spacing, blur leak...

![image](https://github.com/user-attachments/assets/bd3aa8e5-4835-45cb-902e-4958e0462e9a)